api/src/main/java/io/strimzi/api/kafka/KafkaBridgeList.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2018, Strimzi authors.
```

Comment:
I guess we should start to change the year for new files :-)
Check the other new files as well.